ext/couchbase.cxx

```diff
@@ -784,6 +794,9 @@ cb_map_error_code(std::error_code ec, const std::string& message, bool include_e
             case couchbase::errc::key_value::durable_write_re_commit_in_progress:
                 return rb_exc_new_cstr(eDurableWriteReCommitInProgress, what.c_str());
 
+            case couchbase::errc::key_value::mutation_token_outdated:
+                return rb_exc_new_cstr(eMutationTokenOutdated, what.c_str());
+
```

Comment:
`couchbase::errc::range_scan_completed` is not in the switch (the compiler outputs a warning about that) because there is no corresponding Ruby error, it is not exposed to the Ruby-side. Should I map this error to something else?

Reply:
Lets map it to `eBackendError`.